banbot.js

```diff
@@ -6,14 +6,16 @@ require('log-timestamp')
 
 const usage     = 'Commands that Ban Bot can handle:\n' +
                   '_Type a command with no args to get its usage statement_\n' +
-                  '`.quit | .minor | .moderate | .major | .adddays | .rmdays | .oversub | .smurf | .unsuspend`'
+                  '`.quit | .minor | .moderate | .major | .extreme |.adddays | .rmdays | .oversub | .comp | .smurf | .unsuspend`'
 const quit      = '`Usage: .quit <member>`'
 const minor     = '`Usage: .minor <member> [reason]`'
 const moderate  = '`Usage: .moderate <member> [reason]`'
 const major     = '`Usage: .major <member> [reason]`'
+const extreme   = '`Usage: .extreme <member> [reason]`'
 const adddays   = '`Usage: .adddays <member> <number> [reason]`'
 const rmdays    = '`Usage: .rmdays <member> <number> [reason]`'
 const oversub   = '`Usage: .oversub <member>`'
+const comp      = '`Usage: .comp <member>`'
```

Comment:
I'd recommend a `[reason]` optional parameter for a comp penalty. Only the ones that are self-explanatory, like quit, smurf, etc.. don't take a reason.